vaadin-radio-button-flow-demo/src/main/java/com/vaadin/flow/component/radiobutton/demo/RadioButtonGroupView.java

```diff
@@ -15,330 +15,287 @@
-            Image image = new Image("https://vaadin.com/images/vaadin-logo.svg",
-                    "");
-            image.getStyle().set("height", "15px");
-            image.getStyle().set("float", "left");
-            image.getStyle().set("marginRight", "5px");
-            image.getStyle().set("marginTop", "2px");
-            return image;
-        }, Person::getName));
+        // source-example-heading: Entity list
+        RadioButtonGroup<Department> radioGroup = new RadioButtonGroup<>();
+        radioGroup.setLabel("Department");
+        List<Department> departmentList = getDepartments();
+        radioGroup.setItems(departmentList);
+        radioGroup.addThemeVariants(RadioGroupVariant.LUMO_VERTICAL);
         // end-source-example
 
-        group.setId("button-group-icon-generator");
-
-        addCard("Radio button group with icon generator", group);
+        addCard("Entity list", radioGroup);
     }
 
-    private void addDisabled() {
-
+    private void valueChangeEvent() {
         // begin-source-example
-        // source-example-heading: Disabled radio button group
-        RadioButtonGroup<String> group = new RadioButtonGroup<>();
-        group.setItems("foo", "bar", "baz");
-        group.setEnabled(false);
+        // source-example-heading: Value change event
+        RadioButtonGroup<String> radioGroup = new RadioButtonGroup<>();
+        radioGroup.setLabel("Label");
+        radioGroup.setItems("Option one", "Option two", "Option three");
+        radioGroup.addThemeVariants(RadioGroupVariant.LUMO_VERTICAL);
+
+        Div value = new Div();
+        value.setText("Select a value");
+        radioGroup.addValueChangeListener(event -> {
+            if (event.getValue() == null) {
+                value.setText("No option selected");
+            } else {
+                value.setText("Selected: " + event.getValue());
+            }
+        });
         // end-source-example
 
-        group.setId("button-group-disabled");
-
-        addCard("Disabled radio button group", group);
+        addCard("Value change event", radioGroup, value);
     }
 
-    private void addReadOnlyGroup() {
+    private void configurationForReqired() {
         // begin-source-example
-        // source-example-heading: Read-only radio button group
-        Div valueInfo = new Div();
-
-        RadioButtonGroup<String> group = new RadioButtonGroup<>();
-        group.setItems("foo", "bar", "baz");
-        group.setReadOnly(true);
-
-        NativeButton button = new NativeButton("Switch read-only state",
-                event -> group.setReadOnly(!group.isReadOnly()));
-        group.addValueChangeListener(
-                event -> valueInfo.setText(group.getValue()));
+        // source-example-heading: Required
+        Employee employee = new Employee();
+        Binder<Employee> binder = new Binder<>();
+
+        RadioButtonGroup<String> radioGroup = new RadioButtonGroup<>();
+        radioGroup.setLabel("Department");
```

Comment:
Looking at the demo source, "Department" isn't the property being selected here. Should be changed to "Employee title" or just "Title"

Reply:
Done.